README.md

```diff
@@ -1,26 +1,81 @@
 # interception-k2k
+
 All-in-one input key mapper for [Interception Tools](https://gitlab.com/interception/linux/tools).
 
 ## Configuration
 
-User configuration files can be placed under subdirectories of `config`
-directory. Executables will be generated per subdirectory, so you can easily
-manage your rules if you need more instances of `interception-k2k`.
+User configuration files can be placed under subdirectories of `examples` directory, or in a new dir next to it. Executables will be generated per subdirectory, so you can easily manage your rules if you need more instances of `interception-k2k`.
+
+- For simple 1-to-1 mappings, use `map-rules.h.in`.
+  - If you want to disable a key, map it to `KEY_RESERVED`.
+- For many-to-1, use `multi-rules.h.in`.
+- Note that there does not seem to be a way to map a single key input to output multiple keys. Use dual-function-keys for that.
+- For different behavior when a key is tapped and when it's held, use `tap-rules.h.in`.
+
+This repository contains the following example configurations:
+
+### caps2esc
+
+This maps <kbd>caps lock</kbd> to <kbd>esc</kbd> when tapped and to <kbd>left control</kbd> when held.
+
+### ctrl-meta-lock
+
+<kbd>control</kbd> locks when both keys are pressed together. The same for <kbd>meta</kbd>.
+
+### disable-keys
+
+This lets you disable certain keys that you have mapped elsewhere, forcing you to adjust to their new location. If you are chaining configurations, make sure you disable first, then add the new mapping.
+
+### home-row-mods
+
+Mapping <kbd>a</kbd>,<kbd>s</kbd>,<kbd>d</kbd>,<kbd>f</kbd>,<kbd>j</kbd>,<kbd>k</kbd>,<kbd>l</kbd>,<kbd>;</kbd> and <kbd>space</kbd> to <kbd>control</kbd>, <kbd>alt</kbd>, <kbd>meta</kbd> and <kbd>shift</kbd> when held.
+
+### media-keys
+
+<kbd>left meta</kbd> key combinations as media keys.
+
+### qwerty-ws
+
+This remaps the right half of an ANSI laptop keyboard after you have moved around some of its keycaps for a WideSym mod (inspired by [DreymaR's Colemak-CAWS](https://dreymar.colemak.org/ergo-mods.html#symbols)).
+
+### shift2caps
+
+This toggles <kbd>caps lock</kbd> when both <kbd>shift</kbd> keys are pressed together.
+
+### vim-overlay
+
+Holding <kbd>e</kbd> activates vim-like functions on the right side of the keyboard, and holding <kbd>i</kbd> activates some on the left. Note that this mapping seems to be incorrect, so it would need some work before these are actual vim keys. The correct ones look to be commented out.
```

Comment:
> Note that this mapping seems to be incorrect…

Those keys are based on the Dvorak layout so could please mention it instead of saying incorrect?

I'm not sure but maybe it could be renamed to `vim-overlay-dvorak` and/or make a "correct" `vim-overlay-qwert?` copy of it. (Please don't waste your time with the latter if you personally don't use it.)